### DISCUSSION TITLE
Rat genome

### DISCUSSION THREAD

**Discussion by amitpande74** [churchmanlab/genewalk#53] [Q&A] [answered]
Hi,
I want to do analysis of genes in the Rat genome. Is this possible.
Kindly let me know.

regards,
Amit.

**Comment by ri23** [comment]
Hi @amitpande74 ,
We would recommend that you map the rat genes to human genes yourself and then run GeneWalk with the human genes as input list. In this way you can curate yourself that the rat genes you are interested are mapped to the correct human homologs. The mouse genes are also mapped to human orthologs in GeneWalk, but in that case automatically. So if you prefer you could map rat genes to mouse first, but GeneWalk will then still map them to human homologs as well.
Best,
Robert

**Comment by ri23** [comment, marked as answer]
Just fyi: for GeneWalk v1.5.0 and higher: Rat as an organism is now fully automated, similar to GeneWalk for mouse. We allow rat gene identifiers as input gene list with `--id_type rgd_id`. Rat gene ids are then automatically mapped to human ortholog genes with subsequent automated network assembly.

**Reply by amitpande74** [reply]
thanks a lot @ri23
